component-api/src/main/java/org/talend/sdk/component/api/component/InputFinder.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (C) 2006-2021 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.api.component;
+
+import java.io.Serializable;
+import java.util.Iterator;
+import java.util.Map;
+
+import org.talend.sdk.component.api.record.Record;
+
+public interface InputFinder extends Serializable {
```

Comment:
We need some explanation/doc...

Reply:
is it a service, a builtin' service, is it the right place? (needs doc updating maybe)